skxray/fitting/models.py

```diff
@@ -128,7 +128,7 @@ class ElasticModel(Model):
 
     def __init__(self, *args, **kwargs):
         super(ElasticModel, self).__init__(elastic, *args, **kwargs)
-        set_default(self, elastic)
+        #set_default(self, elastic)
```

Comment:
I will delete them. I don't want to set parameters in multiple places. This will cause some bugs.